packages/circuit-ui/components/Toggle/__snapshots__/Toggle.spec.tsx.snap

```diff
@@ -204,21 +205,22 @@ exports[`Toggle should render with default styles 1`] = `
       off
     </span>
   </button>
-  <label
+  <div
```

Comment:
This could break if implementations were trying to style internals using selectors like `.toggle > label`. Within our policy so no need for a major, but should we add a note in the changelog?

Reply:
I've rephrased the changeset to include this information and match the [changelog](https://github.com/sumup-oss/circuit-ui/blob/main/packages/circuit-ui/CHANGELOG.md#600) for the similar change to the `validationHint`s in v6.